src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindAndUpdateIntegrationTest.java

```diff
@@ -455,6 +455,53 @@ public void findByIdUpsert() {
           .body("data.docs[0]", jsonEquals(expected));
     }
 
+    @Test
+    @Order(2)
+    public void findByIdAndColumnUpsert() {
```

Comment:
Based on an earlier comment; would be good to test vs explicit `null` equality: I _think_ (but may be wrong) that explicit `null` property would also be added on insert (when upserting). But even if not we would want to verify expected behavior.

Reply:
Those `return null` is for filters like $exists, $size etc for which data not added to document on upsert. I have added null check filter to the use case to show, null JsonNode value is correctly added to the document.